pkg/roachprod/roachprod.go

```diff
@@ -1406,6 +1406,10 @@ func StartGrafana(
 		// Configure scraping on all nodes in the cluster
 		promCfg.WithCluster(nodes)
 		promCfg.WithNodeExporter(nodes)
+		// Scrape all workload prometheus ports, just in case.
+		for _, i := range nodes {
+			promCfg.WithWorkload(fmt.Sprintf("workload_on_n%d", i), i, 2112)
```

Comment:
Perhaps use this instead? https://github.com/cockroachdb/cockroach/blob/a86fe24ce5849ee74a34ccea1d123fd11661493c/pkg/roachprod/prometheus/prometheus.go#L29